main.py

```diff
@@ -5,15 +5,35 @@
 import ffmpeg
 from ffmpeg import probe
 from ffprobe3 import FFProbe
+import re
 
 # Array of arrays with each internal array representing an expected input resolution.
 # Includes ratios called in confirmation of FFprobe stream request.
 # Gives shortform name for each expected corresponding .png watermark file.
 # .png files are sized the same as SD, HD, Full HD and cropped HD (1440x1080) video
-aspects = [[720,  "4:3",  "SD"],
-          [1280, "16:9",  "HD"],
-          [1920, "16:9", "FHD"],
-          [1440,  "4:3", "CHD"]]
+RESOLUTIONS = {
+  'SD': {
+    'aspect_ratio': '4:3',
+    'width':         720,
+    'full_name':    'Standard definition'
+  },
+  'HD': {
+    'aspect_ratio': '16:9',
+    'width':         1280,
+    'full_name':    'High definition'
+  },
+  'FHD': {
+    'aspect_ratio': '16:9',
+    'width':         1920,
+    'full_name':    'Full high definition'
+  },
+  'CHD': {
+    'aspect_ratio': '4:3',
+    'width':         1440,
+    'full_name':    'Cropped high definition'
+  },
```

Comment:
Here I've changed the list of resolutions to a dict structure rather than nested lists. 

For me, it's easier to read e.g. `RESOLUTIONS['HD']['width']` than `RESOLUTIONS[0][0]`.

I just realised that the `full_name`s aren't used at all though! Opps. I think I intended to use them in a print statement somewhere but then forgot.